holoviews/element/geom.py

```diff
@@ -63,8 +64,27 @@ def _get_selection_expr_for_stream_value(self, **kwargs):
                     (dim(ydim) >= y0) & (dim(ydim) <= y1)
             )
 
-            return selection_expr, bbox
-        return None, None
+            # region_element = Bounds(kwargs['bounds'])
+            x0, y0, x1, y1 = kwargs['bounds']
+            region_element = Curve(([x0, x1, x1, x0, x0], [y0, y0, y1, y1, y0]))
```

Comment:
Why did you switch to a Curve element here? I think now that it's merged I'd probably swap this to `Rectangles`.